WasatchNET/Spectrometer.cs

```diff
@@ -703,7 +705,7 @@ public virtual uint integrationTimeMS
 
                     // logger.debug("setIntegrationTimeMS: {0} ms = lsw {1:x4} msw {2:x4}", ms, lsw, msw);
                     byte[] buf = null;
-                    if (isARM)
+                    if (isARM || isStroker)

```

Comment:
how did you determine that was necessary?

Reply:
I don't know that it is, was having issues with integration time and tried lots of different things, this may have been for debugging pruposes